datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -791,28 +790,72 @@ private void stopServiceSync(final long timeout, final @NonNull TimeUnit unit)
+            // this service and by itself (via eventHandlerStrategyImpl.handleSpaceWarning())
+            unbind();
+        } catch (IllegalArgumentException e) {
+            throw new DataCapturingException(e);
+        } catch (DataCapturingException e) {
+            Log.w(TAG, "Service was either paused or already stopped, so I was unable to unbind from it.");
+        } finally {
+            Log.v(TAG, String.format("Stopping using Intent with context %s", context));
+            Intent stopIntent = new Intent(context, DataCapturingBackgroundService.class);
+            serviceWasActive = context.stopService(stopIntent);
+        }
 
         if (!serviceWasActive) {
-            // The background service was not running so we need to inform the caller of this method ourselves.
-            Intent stoppedBroadcastIntent = new Intent(MessageCodes.BROADCAST_SERVICE_STOPPED);
-            stoppedBroadcastIntent.putExtra(STOPPED_SUCCESSFULLY, false);
-            context.sendBroadcast(stoppedBroadcastIntent);
+            // The background service was *not* running so we need to inform the caller of this method ourselves.
+            // This can happen when we stop a "paused" DataCapturingBackgroundService (which is already stopped).
+            sendServiceStoppedUnsuccessfullyCallback(context);
         }
     }
 
+    /**
+     * This message is sent to the {@link ShutDownFinishedHandler} to inform callers that the async stop
+     * command was executed and that the {@link DataCapturingBackgroundService} was not active when the
+     * stop command was received which can happen when the pause command was executed before.
+     *
+     * @param context The {@link Context} used to send the broadcast from
+     */
+    private void sendServiceStoppedUnsuccessfullyCallback(final Context context) {
+        sendServiceStoppedBroadcast(context, -1, false);
+    }
+
+    /**
+     * This message is sent to the {@link ShutDownFinishedHandler} to inform callers that the async stop
+     * command was executed.
+     *
+     * @param context The {@link Context} used to send the broadcast from
+     * @param measurementIdentifier The id of the stopped measurement if {@code #stoppedSuccessfully}
+     * @param stoppedSuccessfully True if the background service was still alive before stopped
+     */
+    private void sendServiceStoppedBroadcast(final Context context, final long measurementIdentifier,
+                                             final boolean stoppedSuccessfully) {
+        final Intent stoppedBroadcastIntent = new Intent(MessageCodes.FINISHED_HANDLER_BROADCAST_SERVICE_STOPPED);
+        if (stoppedSuccessfully) {
+            Validate.isTrue(measurementIdentifier > 0L);
+            stoppedBroadcastIntent.putExtra(MEASUREMENT_ID, measurementIdentifier);
+        }
+        stoppedBroadcastIntent.putExtra(STOPPED_SUCCESSFULLY, stoppedSuccessfully);
+        context.sendBroadcast(stoppedBroadcastIntent);
```

Comment:
You need to send a local broadcast here or else all apps using the SDK will receive the broadcast (Stadtradeln, Cyface, Cyface Pro).